### ISSUE TITLE
update requirements.txt

### ISSUE THREAD

**Issue by MartinaRuocco** [ascentai/diy-gym#32]
Tried to install it today! Just a couple of modifications to make it work:
1. after running `pip install -r requirements.txt` I got the error:
`"Could not find a version that satisfies the requirement unittest"` 
which I solved by deleting `unittest` from the requirements. Solution explained [here](https://stackoverflow.com/questions/46133254/unittest-installation-error-could-not-find-a-version-that-satisfies-the-requirem).
2. when I tried to run `python ur_high_5.py` I got this other error:
`"segmentation fault (core dumped)"` 
which I solved by installing an old version of gym. 

To summarize I modified the requirements as follows: 
``` 
numpy
gym<=0.15.3
pybullet
pyyaml
tqdm
argparse
requests
```
and now I can launch the test without a problem!
I hope this helps! :)

**Comment by thomascent** [comment]
Hey @MartinaRuocco, thanks for the info! Would you be willing to make a PR to this repo with these changes?

**Comment by MartinaRuocco** [comment]
Already did!
